src/main/java/frc/robot/commands/MoveToHeight.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot.commands;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Elevator;
+
+public class MoveToHeight extends CommandBase {
+  Elevator m_elevator; 
+  double m_elevatorHeight; 
+
+  public MoveToHeight(Elevator elevator, double elevatorHeight) {
+    m_elevator = elevator; 
+    m_elevatorHeight = elevatorHeight; 
+    addRequirements(m_elevator);
+  }
+
+  @Override
+  public void execute() {
+    System.out.println(m_elevator.returnHeightError(m_elevatorHeight)); 
+    m_elevator.set(m_elevator.returnClampedElevatorPID(m_elevatorHeight)); 
+    m_elevator.stopMotorsIfLimitSwitchesTripped(m_elevator.returnClampedElevatorPID(m_elevatorHeight));
+  }
+}
+
+//Note: this command will never stop running, the motor needs to constantly output power to maintain the elevator's height
```

Comment:
Notes like this should be above some relevant code. Also, the way we typically write PID is to have it in the periodic method of the subsystem. So, you can just set the elevator setpoint and the PID will be running in the subsystem.

Reply:
code change added